fuel/transformers/__init__.py

```diff
@@ -164,7 +186,7 @@ def __init__(self, data_stream, iteration_scheme):
             data_stream, iteration_scheme=iteration_scheme)
         self.cache = [[] for _ in self.sources]
 
-    def get_data(self, request=None):
+    def get_data_from_example(self, request=None):
```

Comment:
This should be `get_data_from_batch` (don't forget to pass `batch_input=True` to `super`).